### ISSUE TITLE
Update baseline thickness and dh/dt code

### ISSUE THREAD

**Issue by mankoff** [GEUS-Glaciology-and-Climate/ice_discharge#44]
Current baseline thickness uses a GIMP map with widely varying timestamps among pixels. 

We should:
+ Use a temporally consistent baseline DEM
+ Improve dh/dt treatment with updated dh/dt datasets from either Khan or Winstrup

**Comment by mankoff** [comment]
### Document changes due to PRODEM

+ prodem from d8186bd
+ olddem from ca7b6d8

Comparison is `prodem - olddem` so negative values mean less discharge with the update to prodem.

### Top Few

top frew glaciers appear to change by < +- 1 Gt/year.  The largest absolute changes are from Helheim, Kanger, and ZI. The largest absolute change is ZI at ~0.5 Gt/yr with an absolute discharge of ~15 Gt/yr or a ~3 % change.

![9d2f8520154b729229419763f0ded13f12d397bc](https://github.com/GEUS-Glaciology-and-Climate/ice_discharge/assets/145117/aafa5659-de3d-4957-8e56-bd8e4009d290)

### Regions

Regionally summed discharge is up to -5 Gt/year from CE, NW, and -4 from SE. These are 6 %, 5 %, and ~3 % respectively.

![b68c39d709357aef6d943cbc009e967a7c893ae7](https://github.com/GEUS-Glaciology-and-Climate/ice_discharge/assets/145117/c120952c-2e70-4fb1-b595-99a8a6a0f4aa)

**Comment by mankoff** [comment]
And source code for generating the above plots is:

```python
import matplotlib.pyplot as plt
import numpy as np
import pandas as pd
import xarray as xr

fig = plt.figure(1)
ax = fig.add_subplot(111)

prodem = xr.open_dataset('./PRODEM/out/gate.nc')
olddem = xr.open_dataset('./OLDDEM/out/gate.nc')

LABELS = ['HELHEIMGLETSCHER', 'KANGERLUSSUAQ', 'KOGE_BUGT_C', 'ZACHARIAE_ISSTROM', 'RINK_ISBRAE', 'NIOGHALVFJERDSFJORDEN', 'PETERMANN_GLETSCHER']

prodem = prodem.where(prodem['name_Mouginot'].isin(LABELS), drop=True)
olddem = olddem.where(olddem['name_Mouginot'].isin(LABELS), drop=True)

prodem = prodem.assign_coords({'label': prodem['name_Mouginot'].values})
olddem = olddem.assign_coords({'label': olddem['name_Mouginot'].values})

prodem['d'] = (('label','time'), prodem['discharge'].values)
olddem['d'] = (('label','time'), olddem['discharge'].values)

prodem_df = prodem['d'].to_dataframe().unstack().T
olddem_df = olddem['d'].to_dataframe().unstack().T

(prodem_df - olddem_df).plot(ax=ax)
ax.set_ylim([-2,1])
```

and 

```python
import matplotlib.pyplot as plt
import numpy as np
import pandas as pd
import xarray as xr

fig = plt.figure(1)
fig.clf()
ax = fig.add_subplot(111)

prodem = xr.open_dataset('./PRODEM/out/gate.nc')
olddem = xr.open_dataset('./OLDDEM/out/gate.nc')

prodem = prodem.groupby('region').sum()
olddem = olddem.groupby('region').sum()

prodem_df = prodem['discharge'].to_dataframe().unstack().T
olddem_df = olddem['discharge'].to_dataframe().unstack().T

(prodem_df - olddem_df).plot(ax=ax)
```

**Comment by mankoff** [comment]
Closed by #45
